datamodel/src/main/java/com/hartwig/serve/datamodel/serialization/ServeJson.java

```diff
@@ -0,0 +1,45 @@
+package com.hartwig.serve.datamodel.serialization;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.hartwig.serve.datamodel.RefGenome;
+import com.hartwig.serve.datamodel.ServeRecord;
+
+import org.jetbrains.annotations.NotNull;
+
+public final class ServeJson {
+    private static final ObjectMapper mapper = new ObjectMapper()
+            .registerModule(new GuavaModule())
```

Comment:
Because somehow google collections are used in to the datamodel in de-serialization from json

Reply:
I think you can also add `@Value.Style(jdkOnly = true)` to force immutables to not use guava.